documents/Design.md

```diff
@@ -0,0 +1,25 @@
+# Dataset Viewer Design
+
+Programming Language: Python 3  
+Coding Style: PEP8  
+Design Pattern: Model View Presenter (Passive View)
+## Class Diagram
+![Class Diagram](ClassDiagram.png)  
+DimensionView: Contains the slider, buttons, and stepper for an individual dimension.  
+DimensionPresenter: Manages the behavior of the DimensionView.  
+MainView: Contains the individual DimensionViews and the central plot.  
+MainPresenter: Manages the behavior of the MainView and retrieves data from the MainModel.  
+MainModel: Contains the data array. 
+## Sequence Diagrams
+### Pressing an X Button
```

Comment:
There is no context to this section. How would the reader know that you are not attempting to provide a full set of sequence diagrams for all behaviour?

You should say - the following represents a handful of sequence diagrams to show how the object interplay on a temporal basis to achieve a select set of actions - or something similar